src/videojs-contrib-hls.js

```diff
@@ -134,6 +133,46 @@ export default class HlsHandler extends Component {
       externHls: Hls
     });
 
+    this.masterPlaylistController_.on('loadedmetadata', () => {
+      let audioTracks = this.tech_.audioTracks();
```

Comment:
This is probably too picky (and introduces shadowing), but we have a variable called audioTrackList that is the same thing above as audioTracks is here (same as in different names representing the same thing, though their values will be different).